### ISSUE TITLE
Shueisha (Jump) comic in kobo still can't be DRM removed

### ISSUE THREAD

**Issue by kof11** [noDRM/DeDRM_tools#487]
### Question / bug report

I'm pretty sure this is an old question, I asked it last year and it's still unresolved.
https://i.imgur.com/4LxoG02.jpg
Comics from other publishers can successfully remove DRM. Only Shueisha's comics have problems.
https://i.imgur.com/sj84QqK.jpg
There are a lot of free Shueisha comics here, please prove it's not my problem.
https://books.rakuten.co.jp/event/e-book/free/?l-id=pc-genre-free_seeMore#boy

### Which version of Calibre are you running?

7.4.0

### Which version of the DeDRM plugin are you running?

v10.0.9

### Log output

```log
Paste log output here.
```

**Comment by kof11** [comment]
Jesus, I found the solution myself, use Epubor Ultimate Converter software to successfully remove DRM. It's really solved.

**Comment by ElleKayEm** [comment]
Strange that DeDRM didn't work for you then since Epubor is based on the code here.

**Comment by romad275** [comment]
I’ve had the same result periodically over the years. DeDRM fails but Epubor worked fine. Weird.

Sincerely,
Dennis B. Swaney

Cogito Ergo Mac

**Comment by jeltaqq** [comment]
I don't think DeDRM problems because I'm reading it with other viewers. It is exceptional that there is no xhtml on each pages.

**Comment by js999y** [comment]
@kof11 Does it still work for you on either Calibre / Epubor? I'm having the same issue where DeDRM failed with specifically Shueisha ebooks (converted to gibberish) and when attempting to open error shows
`There is no item named 'OPS/advanced.opf' in the archive`
But Epubor is also failing to convert any of the Shueisha manga - none of the files are openable / status is Failed. Both Calibre and Epubor seem to be blocked right now.

**Comment by ElleKayEm** [comment]
That's an issue with the book.  Can you open it in other ereaders besides calibre's ebook viewer?

**Comment by kof11** [comment]
> @kof11 Does it still work for you on either Calibre / Epubor? 
Still work for me. Here are my steps.
Drag the epub that failed to be converted by Caliber into Epubor, then use Epubor to convert a mobi, then drag the mobi into Caliber, and it will be successful.
I don't know why but it works for me.

**Comment by js999y** [comment]
Yes @ElleKayEm I can open the book in Kobo app with no issue, and confirmed i can read through the whole book with no errors. But when opening with calibre, it seems to be corrupt.

Thank you for giving the steps @kof11 , looks like it only partially works for me. Not sure if it's because the version I'm using or something, but when I try those steps, convert to mobi on Epudor and back to Calibre, half of the manga become missing. I extracted the epub and it misses a chunk in the middle (e.g. 0065.jpg - 0148.jpg missing out of 200 page book where 1 img = 1 page). Can I ask what version of Epubor / Calibre you're using?

**Comment by js999y** [comment]
Thanks @ElleKayEm Just gave that a go. Opening the epub file downloaded from Calibre on other viewers do not work, it just show blank pages. But from unzipping epub file copied from Calibre, it looks like you're right that there are no issue with DeDRM. The only thing that seems different are file structure on unzipped epub,
File structure for a book w issue loading on viewer --
`
├── images
│   ├── <first_page>.jpg
│   ...
│   └── <last_page>.jpg
├── navigation_standard.ncx
├── navigation_standard.xhtml
└── standard.opf
`
File structure for a book w/o issue loading on viewer --
`├── item
│   ├── image
│   │   ├── <first_page>.jpg
│   ...
│   │   └── <last_page>,jpg
│   ├── navigation-documents.ncx
│   ├── navigation-documents.xhtml
│   ├── standard.opf
│   ├── style
│   │   └── fixed-layout-jp.css
│   └── xhtml
│       ├── <first>.xhtml
│   ...
│       └── <last>.xhtml
`
Probably something to do with navigation but the images are all there without missing any, so just had to compile the images back to pdf then load it to Calibre and convert file type to epub. Thanks again!

**Comment by jason3232** [comment]
I had encountered this issue as well and (I think) I have an explanation in this https://github.com/noDRM/DeDRM_tools/issues/769#issuecomment-2716879067

tl;dr the decrypted epub file don't have any xhtml/html files and the TOC directly link into images which aren't standard epub
